jiant/tasks/lib/templates/squad_style/core.py

```diff
@@ -310,13 +333,19 @@ def __init__(
         n_best_size=20,
         max_answer_length=30,
         null_score_diff_threshold=0.0,
+        doc_stride=128,
+        max_query_length=64,
```

Comment:
Can you specify where these come from?

Reply:
These come from the defaults in Hugging Face's SQuAD script.